src/swedish-pancakes.md

```diff
@@ -0,0 +1,49 @@
+# Swedish Pancakes
+
+![Swedish Pancakes](pix/swedish-pancakes.webp)
```

Comment:
I think the picture format is `<name-of-recipe>-00.webp` for recipes with multiple images.